### ISSUE TITLE
listing of animes with 001 doesnt work!

### ISSUE THREAD

**Issue by krishne35** [ssnjrthegr8/stremio-gdrive#9]
the drive just works fine with s01E01 type of content but when it comes to onepeice where al of them are listed as 001 002 003 the addon just neglects them 

The torrentio works fine so something on the gdrive addon side maybe?

**Comment by krishne35** [comment]
Torentio:
![image](https://user-images.githubusercontent.com/35328770/135741268-1e7b68a0-05bb-4a63-a0f1-648df6a2815c.png)
Stremio-gdrive:
![image](https://user-images.githubusercontent.com/35328770/135741279-c3905bf9-532b-4465-bdec-e1065b04aef2.png)

**Comment by krishne35** [comment]
basicaly all the episodes need to be remped to S0XE0X for Stremio-Drive to get the links

**Comment by ssnjrthegr8** [comment]
It might be due to stremio gdrive using imdb api instead of cinemeta by default.

To alleviate these fringe cases I will be implementing a custom query feature where you can input the name and season/year.

This will take a while since I haven't got any free time and my pc is not working properly. So I cant really give an eta for the fix.

**Comment by krishne35** [comment]
Manually inputting season and year ???

**Comment by theghost970** [comment]
I believe the plugin "Anime Kitsu" is used for Anime catalogue, not Cinemata

**Comment by devansh-hehehe** [comment]
> It might be due to stremio gdrive using imdb api instead of cinemeta by default.
> 
> To alleviate these fringe cases I will be implementing a custom query feature where you can input the name and season/year.
> 
> This will take a while since I haven't got any free time and my pc is not working properly. So I cant really give an eta for the fix.



> It might be due to stremio gdrive using imdb api instead of cinemeta by default.
> 
> To alleviate these fringe cases I will be implementing a custom query feature where you can input the name and season/year.
> 
> This will take a while since I haven't got any free time and my pc is not working properly. So I cant really give an eta for the fix.

Yeah I too a facing this problem.... Owner If you are "Now" Not busy, Please Fix this.... I also have a file named
Black clover - E01 and Gdrive addon doesn't search for it.

**Comment by krishne35** [comment]
> Yeah I too a facing this problem.... Owner If you are "Now" Not busy, Please Fix this.... I also have a file named
> Black clover - E01 and Gdrive addon doesn't search for it. 

What's with the attitude mate it's Free project don't demand for things to be fixed!!

**Comment by devansh-hehehe** [comment]
> > Yeah I too a facing this problem.... Owner If you are "Now" Not busy, Please Fix this.... I also have a file named
> > Black clover - E01 and Gdrive addon doesn't search for it.
> 
> What's with the attitude mate it's Free project don't demand for things to be fixed!!

I don't know where in India you belong to, But If specifically wrote "Now" it meant that there's 50-50% probability that he may or may not do it.... No Words in my comments Demands Him to do IT... Neither he is my employee.... Its just a simple request put to the Smart guy by a  guy who is requesting him to do me some favour...

**Comment by devansh-hehehe** [comment]
> > Yeah I too a facing this problem.... Owner If you are "Now" Not busy, Please Fix this.... I also have a file named
> > Black clover - E01 and Gdrive addon doesn't search for it.
> 
> What's with the attitude mate it's Free project don't demand for things to be fixed!!

I Demand You TO "TAKE THINGS POSITIVELY" negative earthly creature.

**Comment by devansh-hehehe** [comment]
> > > Yeah I too a facing this problem.... Owner If you are "Now" Not busy, Please Fix this.... I also have a file named
> > > Black clover - E01 and Gdrive addon doesn't search for it.
> > 
> > 
> > What's with the attitude mate it's Free project don't demand for things to be fixed!!
> 
> I Demand You TO "TAKE THINGS POSITIVELY" negative earthly creature.

Point to be noted :- This Ain't a request 😡😡
